common/Resources/ti.internal/extensions/node/internal/util/inspect.js

```diff
@@ -0,0 +1,1695 @@
+	if (ctx.showHidden) {
+		output.push(`[size]: ${ctx.stylize(`${value.size}`, 'number')}`);
+	}
+	return output;
+}
+
+function formatMap(ctx, value, recurseTimes) {
+	const output = [];
+	ctx.indentationLvl += 2;
+	for (const [ k, v ] of value) {
+		output.push(`${formatValue(ctx, k, recurseTimes)} => ${formatValue(ctx, v, recurseTimes)}`);
+	}
+	ctx.indentationLvl -= 2;
+	// See comment in formatSet
+	if (ctx.showHidden) {
+		output.push(`[size]: ${ctx.stylize(`${value.size}`, 'number')}`);
+	}
+	return output;
+}
+
+function formatSetIterInner(ctx, recurseTimes, entries, state) {
+	const maxArrayLength = Math.max(ctx.maxArrayLength, 0);
+	const maxLength = Math.min(maxArrayLength, entries.length);
+	let output = new Array(maxLength);
+	ctx.indentationLvl += 2;
+	for (var i = 0; i < maxLength; i++) {
+		output[i] = formatValue(ctx, entries[i], recurseTimes);
+	}
+	ctx.indentationLvl -= 2;
+	if (state === kWeak && !ctx.sorted) {
+		// Sort all entries to have a halfway reliable output (if more entries than
+		// retrieved ones exist, we can not reliably return the same output) if the
+		// output is not sorted anyway.
+		output = output.sort();
+	}
+	const remaining = entries.length - maxLength;
+	if (remaining > 0) {
+		output.push(`... ${remaining} more item${remaining > 1 ? 's' : ''}`);
+	}
+	return output;
+}
+
+function formatMapIterInner(ctx, recurseTimes, entries, state) {
+	const maxArrayLength = Math.max(ctx.maxArrayLength, 0);
+	// Entries exist as [key1, val1, key2, val2, ...]
+	const len = entries.length / 2;
+	const remaining = len - maxArrayLength;
+	const maxLength = Math.min(maxArrayLength, len);
+	let output = new Array(maxLength);
+	let i = 0;
+	ctx.indentationLvl += 2;
+	if (state === kWeak) {
+		for (; i < maxLength; i++) {
+			const pos = i * 2;
+			output[i] = `${formatValue(ctx, entries[pos], recurseTimes)}`
+				+ ` => ${formatValue(ctx, entries[pos + 1], recurseTimes)}`;
+		}
+		// Sort all entries to have a halfway reliable output (if more entries than
+		// retrieved ones exist, we can not reliably return the same output) if the
+		// output is not sorted anyway.
+		if (!ctx.sorted) {
+			output = output.sort();
+		}
+	} else {
+		for (; i < maxLength; i++) {
+			const pos = i * 2;
+			const res = [
+				formatValue(ctx, entries[pos], recurseTimes),
+				formatValue(ctx, entries[pos + 1], recurseTimes)
+			];
+			output[i] = reduceToSingleString(
+				ctx, res, '', [ '[', ']' ], kArrayExtrasType, recurseTimes);
+		}
+	}
+	ctx.indentationLvl -= 2;
+	if (remaining > 0) {
+		output.push(`... ${remaining} more item${remaining > 1 ? 's' : ''}`);
+	}
+	return output;
+}
+
+function formatWeakCollection(ctx) {
+	return [ ctx.stylize('<items unknown>', 'special') ];
+}
+
+function formatWeakSet(ctx, value, recurseTimes) {
+	// Node calls into native to get a preview of actual values which we can't do
+	return formatWeakCollection(ctx);
+}
+
+function formatWeakMap(ctx, value, recurseTimes) {
```

Comment:
<!--
  0 failure: 
  2 warning:  common/Resources/..., common/Resources/...
  
  
  DangerID: danger-id-Danger;
  File: common/Resources/ti.internal/extensions/node/internal/util/inspect.js;
  Line: 1432;
-->

- :warning: common/Resources/ti.internal/extensions/node/internal/util/inspect.js line 1432 – 'recurseTimes' is defined but never used. Allowed unused args must match /^_.+/u. (no-unused-vars)
- :warning: common/Resources/ti.internal/extensions/node/internal/util/inspect.js line 1432 – 'value' is defined but never used. Allowed unused args must match /^_.+/u. (no-unused-vars)